partials/content-single.php

```diff
@@ -33,8 +33,8 @@
 			$content = apply_filters( 'the_content', get_the_content() );
 			echo $content;
 		}
-		global $multipage;
-		if ( $multipage ) {
+        global $multipage;
+		if ( $multipage && $content_only === false ) {
```

Comment:
Not sure if this is strictly needed here? I think the nav element is actually generated in footer.php, but not 100% sure